recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "gensim" %}
+{% set version = "2.3.0" %}
+{% set sha256 = "7d0dccc7d2c576e270037949874800b7cfbc86ef081ff981483f612cd18e223f" %}
```

Comment:
If a new release has been made when this recipe hasn't been merged, we update it the two lines from here to ensure that the latest version is packaged.

Otherwise, if this patch is merged before the new release, we will be sending a pull request to the feedstock repository of gensim, which would look similar to this- https://github.com/conda-forge/validictory-feedstock and the maintainers of the recipe(mentioned in the recipe-maintainers section of the recipe) will take a look and merge the patch to update the version of the conda package.

The ``sha256`` comes from the project link in [pypi.io](https://pypi.org/project/gensim/#files).